server/src/game/objects/player.ts

```diff
@@ -1303,6 +1303,8 @@ export class Player extends BaseGameObject {
             }
         }
 
+        if (this.hasPerk("bonus_assault")) finalDamage *= 1.08;
```

Comment:
pretty sure that only applied to bullets and should be done in bullet.ts by checking if the trail is saturated